README.md

```diff
@@ -60,73 +60,37 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> First Microverse Project.
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
```

Comment:
- [ ] Well done using Microverse template as required however, I recommend deleting Microverse title and logo and replace them with the title of your project.